students/Lincoln_Zhang/Session2/grid_print.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+#Session 2 Class excercise
+#print grid like this
+
+print("Please make a script generating grid like this.")
+
+print("""
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+""")
+
+boxsize = int(input("Please input box size: "))
+gridsize = int(input("Please input grid size: "))
+
+def horizontal_line(boxsize):
+	return "- " * boxsize + "+ "
+
+def vertical_line(boxsize):
+	return "  " * boxsize + "| " 
+
+# def grid_print_1(boxsize, gridsize):
```

Comment:
We will still want to get grid_print_1 working. The initial print_grid should always create a grid of 4 cells with the total height/length being the parameter we give it. For example: print_grid(9) should return a 4x4 grid where the entire length/height (2 cells worth) is 9. Hint: The middle '+' will handle it such that the each individual cells are of 4 length each.

Reply:
A commit for this feedback is pushed